src/Raven.Studio/wwwroot/index.html

```diff
@@ -60,6 +60,10 @@
         <script type="text/javascript" src="lib/knockout-delegatedEvents/build/knockout-delegatedEvents.js"></script>
         <script type="text/javascript" src="lib/knockout-postbox/build/knockout-postbox.js"></script>
         <script type="text/javascript" src="lib/moment/moment.js"></script>
+        <script type="text/javascript" src="lib/cronstrue/dist/cronstrue.min.js"></script>
```

Comment:
Please don't use minified version here. We minify bundles for release, but having full version is easier to debugging.